tests/unit/models/package-info-cache-test.js

```diff
@@ -0,0 +1,219 @@
+'use strict';
+
+const path = require('path');
+const expect = require('chai').expect;
+const Project = require('../../../lib/models/project');
+let addonFixturePath = path.resolve(__dirname, '../../fixtures/addon');
+const MockUI = require('console-ui/mock');
+const MockCLI = require('../../helpers/mock-cli');
+
+describe('models/package-info-cache.js', function() {
+  let project, projectPath, packageJsonPath, packageContents, projectPackageInfo, ui, cli, pic;
+  this.timeout(100000);
+
+  beforeEach(function() {
+    ui = new MockUI();
+    cli = new MockCLI({ ui });
+  });
+
+  describe('packageInfo contents tests on valid project', function() {
+    beforeEach(function() {
+      projectPath = path.resolve(addonFixturePath, 'simple');
+      packageJsonPath = path.join(projectPath, 'package.json');
+      packageContents = require(packageJsonPath);
+      project = new Project(projectPath, packageContents, ui, cli);
+      pic = project._packageInfoCache;
```

Comment:
given that this is used in tests, should we make `project.packageInfoCache` instead? And annotate its API doc to be `@protected`? It feels a tad funky to reach into private stuff, to test.

Reply:
That sounds like a good idea. Same for addon.